### PR TITLE
Updated documentation for config file from local file system

### PR DESCRIPTION
**Pull request by remmelt**
I was looking for a way to supply a config file from the local fs instead of having to host it somewhere. I found how to do it in a closed issue, it would be great if that could be in the regular documentation.

Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ To use your own config file, set the `LOGSTASH_CONFIG_URL` environment variable
       -p 9200:9200 \
       pblittle/docker-logstash
 
+To use a config file from the local file system, mount the file as a volume using the `-v` option. Replace `/my/logstash.conf` with the absolute path to your config file.
```
Comment (pblittle) [inline README.md]: @remmelt, to keep the documentation uniform, please use the `<your_whatever>` syntax. Something like `<your_logstash_config>`.

File: README.md [hunk 2]
```diff
@@ -27,6 +27,14 @@ To use your own config file, set the `LOGSTASH_CONFIG_URL` environment variable
...
+
+    $ docker run -d \
+      -v /my/logstash.conf:/opt/logstash.conf \
```
Comment (pblittle) [inline README.md]: @remmelt, I have moved the config location to `/opt/logstash/conf.d`. This change removes the very bad `/opt/logstash.conf` restriction. Now any file in `/opt/logstash/conf.d` will get loaded by logstash.

**Comment by remmelt** [comment]
Thank you for the feedback!

I have updated the docs.

**Comment by pblittle** [comment]
@remmelt, LGTM. Thanks for your contribution. I really appreciate it.

**Pull request merged**